libs/utils/image-video-link.js

```diff
@@ -1,7 +1,7 @@
 import { getConfig, loadStyle, createTag } from './utils.js';
 
-const PLAY_ICON_SVG = `<svg xmlns="http://www.w3.org/2000/svg" width="24" height="32" viewBox="0 0 24 32" fill="none" class="play-icon">
-                        <path d="M24 16.0005L0 32L1.39876e-06 0L24 16.0005Z" fill="white"/>
+const PLAY_ICON_SVG = `<svg xmlns="http://www.w3.org/2000/svg" width="40" height="42" viewBox="0 0 40 42" fill="none" class="play-icon">
```

Comment:
Since this has no `title`, is it decorative? If so, let's add a `aria-hidden="true"` attribute.